packages/uicore/src/components/MultiSelectDropDown/MultiSelectDropDown.tsx

```diff
@@ -51,6 +51,7 @@ export interface MultiSelectDropDownProps
   hideItemCount?: boolean
   allowSearch?: boolean
   onPopoverClose?(opts: MultiSelectOption[]): void
+  customItemRender?: (item: MultiSelectOption) => JSX.Element
```

Comment:
let's just call it `itemRenderer` instead of `customItemRender`.

see `itemRenderer` docs for Select component here: https://blueprintjs.com/docs/#select/select-component